frontend/src/components/dating/DatingInterestTag.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Checkbox } from "@chakra-ui/react"
+import { Dispatch, FC, SetStateAction } from "react"
+
+const DatingInterestTag: FC<{
+    interestId: string
+    interestName: string
+    onOpen: () => void
+    selectedInterests: String | String[]
+    numOfInterest: number
+    setNumOfInterest: Dispatch<SetStateAction<number>>
+    setSelectedInterest: Dispatch<any>
+}> = ({ interestId, interestName, onOpen, selectedInterests, numOfInterest, setNumOfInterest, setSelectedInterest }) => {
+    // Check if interestId is in the selectedInterest state or not
+    function idExists(interestId: string) {
+        for (var i = 0; i < selectedInterests.length; i++) {
```

Comment:
อันนี้ก็ var ถ้า scoop ใน for loop ใช้ let ก็พอ

Reply:
เราแก้ตรงนี้ใหม่ละนะๆ